Bottom: ad1fc608089812922f4e37f7b7e3c4362af6e30e
Top:    9c8d4f5bea2d9c2fa1a47c2728940ca938ee89ad
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 15:58:58 +0000

Refresh of extend-object3d-code-to

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index af855b8..162eb64 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -302,6 +302,7 @@ get_contour_edge_n_round_geometry_in_step_mm (PLINE *contour, int n, double *cx,
       *cw = (contour->Flags.orient != PLF_DIR);
       return;
     }
+#endif
 
   while (n > 0) {
     edge = edge->next; /* The VNODE structure is circularly linked, so wrapping is OK */
